libs/types/src/auth/auth.ts

```diff
@@ -79,14 +79,17 @@ export const DEFAULT_STARTING_CARD_LOCKOUT_DURATION_SECONDS: StartingCardLockout
  * The inactive/idle session time limit, after which the user must reauthenticate - a VVSG2
  * requirement
  */
-export type InactiveSessionTimeLimitMinutes = 10 | 15 | 20 | 25 | 30;
+export type InactiveSessionTimeLimitMinutes = 10 | 15 | 20 | 30 | 360;
 export const InactiveSessionTimeLimitMinutesSchema: z.ZodSchema<InactiveSessionTimeLimitMinutes> =
   z.union([
     z.literal(10),
     z.literal(15),
     z.literal(20),
-    z.literal(25),
```

Comment:
`25` felt like a somewhat random option to me, so I just dropped it hah.

I was thinking of dropping the stricter validation altogether such that the data model supports any positive integer value. But the validation feels helpful for now, since we're editing system settings files manually and not through a UI (i.e. VxDesign).